commit afa571e5c76063af71af626415191589b53d042f (refs/changes/37/3037/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2017-12-05T15:20:49+00:00 (1 year, 10 months ago)
    
    OS-6487 ltp: open12 fails acquiring device
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Approved by: Dan McDonald <danmcd@joyent.com>

diff --git a/usr/src/lib/brand/lx/testing/ltp_skiplist b/usr/src/lib/brand/lx/testing/ltp_skiplist
index 09be284e0c..09ff2cacc1 100644
--- a/usr/src/lib/brand/lx/testing/ltp_skiplist
+++ b/usr/src/lib/brand/lx/testing/ltp_skiplist
@@ -59,6 +59,7 @@ move_pages11
 mremap04		# we don't yet support remapping shm
 open01
 open11
+open12			# needs a device
 ptrace04
 quotactl01
 quotactl02
